OpenSim/Sandbox/MatlabScripts/createActuatorsFile.m

```diff
@@ -0,0 +1,156 @@
+               % get the coordainte motion type
+               motion = char(coordinate.getMotionType);
+               % to get the axis value for the coordiante, we need to drill
+               % down into the coordinate transform axis
+               eval(['concreteJoint = ' char(joint.getConcreteClassName()) '.safeDownCast(joint);'])
+               sptr = concreteJoint.getSpatialTransform;
+               for ip = 0 : 5
+                  if strcmp(char(sptr.getCoordinateNames().get(ip)), char(coordinate.getName))
+                        sptr.getTransformAxis(ip).getAxis(axisValues);
+                        break
+                  end
+               end
+
+
+               % make a torque actuator if a rotational coordinate
+               if strcmp(motion, 'Rotational')
+                   newActuator = TorqueActuator(joint.getParentFrame(),...
+                                         joint.getParentFrame(),...
+                                         axisValues,...
+                                         1);
+
+               % make a point actuator if a translational coordainte.
+             elseif strcmp(motion, 'translational')
+                    % make a new Point actuator
+                    newActuator = PointActuator();
+                    % set the body
+                    newActuator.set_body(char(joint.getChildFrame().getName()))
+                    % set <point>      -0.07243760       0.00000000       0.00000000 </point>
+                    newActuator.set_point(massCenter)
+                    % set <point_is_global> false </point_is_global>
+                    newActuator.set_point_is_global(0)
+                    % set <direction>      -0.00000000       1.00000000      -0.00000000 </direction>
+                    newActuator.set_direction(axisValues)
+                    % set <force_is_global> true </force_is_global>
+                    newActuator.set_force_is_global(1)
+              else % something else that we don't support right now
+                    error(['Motion Type ' char(motion) 'not supported yet'])
+              end
+        else % if the joint type is not free or custom, just add coordinate actuators
+                % make a new coordinate actuator for that coordinate
+                newActuator = CoordinateActuator();
+        end
+
+    else % the coordinate is not connected to ground, and can just be a
+         % coordinate actuator.
+         newActuator = CoordinateActuator( char(coordinate.getName) );
+    end
+
+    % set the optimal force for that coordinate
+    newActuator.setOptimalForce(optimalForce);
+    % set the actuator name
+    newActuator.setName( coordinate.getName() );
+    % set the optimal force for that coordinate
+    newActuator.setOptimalForce(optimalForce);
+    % set min and max controls
+    newActuator.setMaxControl(Inf)
+    newActuator.setMinControl(-Inf)
+
+    % append the new acuator onto the empty force set
```

Comment:
From memory, `Append` used to cause Matlab crashing. When should we use `Append`, `cloneAndAppend`? This seems confusing.